### PR TITLE
 Se arregla bug del icono de la pwa, se arregla bug de direccion del …

### PR DESCRIPTION
**Pull request by ssantss**
…manifest, se cambia el short name ah uno mas generico

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "intl-tel-input": "^16.0.2",
-    "jquery": "^3.4.1"
+    "jquery": "^3.4.1",
+    "serve": "^13.0.2"
```
Comment (maoaiz) [inline package.json:15]: Serve is not a production dependency, it should be placed in the `devDependencies`

**Review by maoaiz — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (maoaiz) [reply package.json:15]: Or it should be on your global packages

**Comment by ssantss** [comment]
changes finished, css is fixed so that it is more functional in mobile version

**Review by maoaiz — APPROVED** [review]
Cool! Thanks

**Pull request closed without merging**